peoplefinder/templates/peoplefinder/team.html

```diff
@@ -73,6 +73,9 @@ <h2 class="govuk-heading-m">Teams within {{ team.short_name }}</h2>
             </ul>
         {% else %}
             <h2 class="govuk-heading-m">People in {{ team.short_name }}</h2>
+            {% if members %}
+                <p class="govuk-body"><a class="govuk-link govuk-link--no-visited-state" href="#" hx-get="{% url 'team-email-members' team.slug %}" hx-target="closest p">Get email all members link</a></p>
```

Comment:
```suggestion
                <p class="govuk-body"><a class="govuk-link govuk-link--no-visited-state" href="#" hx-get="{% url 'team-email-members' team.slug %}" hx-target="closest p">Get link to email all members</a></p>
```

Just copied above for convenience.  Might be better with a more action oriented bit of copy still, e.g. "Email all members" ??

Reply:
I struggled on the copy because of how the mechanism work. Let's discuss offline.